src/main/assets/cloudfront_function/error.js

```diff
@@ -0,0 +1,8 @@
+function handler(event) {
+    var request = event.request;
+    
```

Comment:
```suggestion
          if (request.uri.endsWith('/')) {
          request.uri += 'index.html';
      }
```
check with frontend devs if they approve

Reply:
I'll try to apply to see if work as expected

Reply:
Tested and keep working as expected